src/ray/common/cgroup/cgroup_setup.cc

```diff
@@ -248,8 +248,8 @@ Status CgroupSetup::InitializeCgroupV2Directory(const std::string &directory,
   // Check cgroup accessibility before setup.
   RAY_RETURN_NOT_OK(internal::CheckCgroupV2MountedRW(directory));
 
-  // Check cgroup subtree control before setup.
-  if (Status s = internal::CheckBaseCgroupSubtreeController(directory); !s.ok()) {
+  // Check cgroup available controllers before setup.
```

Comment:
Before any cgroup operations, we should check available controllers, and enable subtree controllers.